### PR TITLE
Fix issue with using getInt instead of asDouble for JSI

### PR DESCRIPTION
**Pull request by jonthysell**
JSI makes all dynamic numbers doubles, so we can't use getInt - we have to asDouble and cast it to int.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3094)

### CODE REVIEW

File: vnext/ReactUWP/Views/FrameworkElementViewManager.cpp [hunk 1]
```diff
@@ -335,7 +335,7 @@ void FrameworkElementViewManager::UpdateProperties(
         AnnounceLiveRegionChangedIfNeeded(element);
       } else if (propertyName == "accessibilityPosInSet") {
         if (propertyValue.isNumber()) {
-          auto value = static_cast<int>(propertyValue.getInt());
+          auto value = static_cast<int>(propertyValue.asDouble());
```
Comment (Khalef1) [inline vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: >auto [](start = 10, length = 4)

Not directly related to your CR, but do you know why we have properties for "position in set" and "size", but not "level"? These 3 go together to express a list and the indentation level of the list members.

File: vnext/ReactUWP/Views/FrameworkElementViewManager.cpp [hunk 2]
```diff
@@ -347,7 +347,7 @@ void FrameworkElementViewManager::UpdateProperties(
         }
       } else if (propertyName == "accessibilitySetSize") {
         if (propertyValue.isNumber()) {
-          auto value = static_cast<int>(propertyValue.getInt());
+          auto value = static_cast<int>(propertyValue.asDouble());
```
Comment (Khalef1) [inline vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: >propertyValue.asDouble() [](start = 40, length = 24)

Also check that it is less than INT_MAX before the cast (here and on line 338)

File: vnext/ReactUWP/Views/FrameworkElementViewManager.cpp (hunk 1 above)
Comment (jonthysell) [reply vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: We plan on pushing these properties back up into RN, so we followed the aria specifications to be more cross-platform and less windows-specific, which only includes posInSet and setSize.

Comment (Khalef1) [reply vnext/ReactUWP/Views/FrameworkElementViewManager.cpp]: There is an aria-level property that exists. Doesn't it map to the level property in UIA?

**Pull request merged**